app/assets/javascripts/initializers/initScrolling.js.erb

```diff
@@ -166,8 +166,37 @@ function insertArticles(articles) {
   });
   var distanceFromBottom = document.documentElement.scrollHeight - document.body.scrollTop;
   var newNode = document.createElement("div");
+  newNode.classList.add('paged-stories')
```

Comment:
missing trailing semi colon

Reply:
Thanks! Annoying javascript linting doesn't work in `erb` files!